### ISSUE TITLE
[Regression] Peculiar artifacts in Bloodborne (CUSA03173) after #933

### ISSUE THREAD

**Issue by lukaszarczynski** [shadps4-emu/shadPS4#1075]
System:
OS: Windows 10
CPU: AMD Ryzen 7 5800X3D
GPU: AMD Radeon RX 7900 XT (24.20.11.01-240902a-407173E-KB-AMD-Software-Adrenalin-Edition)
RAM: 32GB

After merging [#933](https://github.com/shadps4-emu/shadPS4/pull/933), Bloodborne shows some peculiar artifacts. Trying to resize the window crashes the emulator
[shad_log.txt](https://github.com/user-attachments/files/17137849/shad_log.txt)
![20240925205053_1](https://github.com/user-attachments/assets/0644758d-ca76-4167-9f96-dda7ded613b4)
![20240925205059_1](https://github.com/user-attachments/assets/c6ce13d1-1410-49cf-be1a-ddba40cb90e2)
![20240925205114_1](https://github.com/user-attachments/assets/bca9c212-4ed5-42ab-bb53-e2b0687ed5d8)
![20240925212600_2](https://github.com/user-attachments/assets/b30557a1-7130-4481-86cf-70f27e75095b)

**Comment by R1chterScale** [comment]
Also experienced on Linux, both with the AMDPRO drivers and with RADV (same GPU)

**Comment by StevenMiller123** [comment]
Duplicate of https://github.com/shadps4-emu/shadPS4/issues/1070

**Comment by psucien** [comment]
Should be fixed with 6a70d08
